README.md

```diff
@@ -47,3 +47,23 @@ Decode a binary:
             97,42,106,106>>,
     bert:decode(Bin).
     > #{key => [{some, "tuple"}, {true, 42}]}
+
+## Documentation generation
+
+### Edoc
+
+#### Generate public API
+```
+rebar3 edoc
+```
+
+#### Generate private API
+```
+rebar3 as edoc_private edoc
```

Comment:
When I run this I get: "No entry for profile edoc_private in config" I guess `rebar.config` needs a profile entry for that?

Reply:
> When I run this I get: "No entry for profile edoc_private in config" I guess `rebar.config` needs a profile entry for that?

Yes. You ar right. This is my oversight. Thank you for bringing this to my attention. Made the appropriate changes.